glide.yaml

```diff
@@ -0,0 +1,18 @@
+package: .
+homepage: https://github.com/HewlettPackard/oneview-golang
+import:
+- package: github.com/docker/machine
+  version: v0.8.0-rc2
+  subpackages:
+  - libmachine/log
+- package: github.com/stretchr/testify
+  version: v1.1.3
```

Comment:
Did you want to use [version ranges](http://glide.readthedocs.io/en/latest/versions/)?